{"uuid":"87d4567a-7446-4cb0-81bd-36f353194dce","historyId":"56dd93696e0c553e3c9806c88aa8a39","fullName":"test.ProductsTests.ValidProductsTests.deleteProduct_Valid","labels":[{"name":"package","value":"test.ProductsTests.ValidProductsTests"},{"name":"testClass","value":"test.ProductsTests.ValidProductsTests"},{"name":"testMethod","value":"deleteProduct_Valid"},{"name":"parentSuite","value":"All Tests Suite"},{"name":"suite","value":"All Tests"},{"name":"subSuite","value":"test.ProductsTests.ValidProductsTests"},{"name":"host","value":"DESKTOP-M2GQIBG"},{"name":"thread","value":"12268@DESKTOP-M2GQIBG.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"deleteProduct_Valid","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [195] but found [1]","trace":"java.lang.AssertionError: expected [195] but found [1]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1590)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:1431)\r\n\tat org.testng.Assert.assertEquals(Assert.java:1395)\r\n\tat org.testng.Assert.assertEquals(Assert.java:1441)\r\n\tat test.ProductsTests.ValidProductsTests.deleteProduct_Valid(ValidProductsTests.java:135)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\r\n\tat org.testng.TestRunner.run(TestRunner.java:616)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","steps":[],"attachments":[{"name":"Test Failure Details","source":"3d7719df-25d9-4f6e-89c4-6989a2f3706f-attachment","type":"text/plain"}],"parameters":[],"start":1755450343260,"stop":1755450343666}